tests/run/apply-line.scala

```diff
@@ -0,0 +1,26 @@
+
+object Test:
```

Comment:
You'll need a
```scala
// scalajs: --skip
```
at the top of this file.

Reply:
aha!